rvip/package.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<package format="2">
+  <name>rvip</name>
+  <version>0.0.1</version>
+  <description>The rvip package</description>
+  <author>Bey Hao Yun</author>
+  <maintainer email="beyhy@artc.a-star.edu.sg">Bey Hao Yun</maintainer>
+
+  <license>BSD</license>
+
+  <buildtool_depend>catkin</buildtool_depend>
+
+  <depend>cv_bridge</depend>
+  <depend>geometry_msgs</depend>
+  <depend>pcl_ros</depend>
+  <depend>roscpp</depend>
+  <depend>sensor_msgs</depend>
+  <depend>std_msgs</depend>
+  <depend>tf</depend>
+  <depend>visualization_msgs</depend>
+
+  <build_depend>image_transport</build_depend>
```

Comment:
I'm not sure this should not also be a `depend` instead.

Reply:
Addressed in recent commits.